### DISCUSSION TITLE
Supabase form

### DISCUSSION THREAD

**Discussion by vinliao** [supabase/supabase#5995] [Feature Requests]
# Feature request

## Is your feature request related to a problem? Please describe.

Airtable allows their users to create form out of a table (ex: https://youtu.be/h09yz1D05UI). Wouldn't it be nice if Supabase can also generate a form to insert data directly to a table?

## Describe the solution you'd like

The form can be served by `<supabase-url>.supabase.co/<table-name>/form` or something like that. Supabase users can then create a table, generate the form, share the link, and collect data without touching any code.

## Additional context

Supabase is great. I love it.

**Comment by kiwicopple** [comment]
Great idea @vinliao. I actually saw this morning that @zernonia is working on this himself (https://twitter.com/zernonia/status/1505811526355386368)

This is probably something we will support in the future - we'll add it to the backlog for now.

I will move this one into a discussion for now. If anyone has any ideas on how to make this great then feel free to comment with your ideas
